src/lkl/setup.c

```diff
@@ -1385,42 +1376,6 @@ void lkl_start_init()
     // Provide LKL host ops and virtio block device ops
     lkl_host_ops = sgxlkl_host_ops;
 
-    // TODO Make tracing options configurable via SGX-LKL config file.
-    if (getenv_bool("SGXLKL_TRACE_SYSCALL", 0))
-    {
-        sgxlkl_trace_lkl_syscall = 1;
-        sgxlkl_trace_internal_syscall = 1;
-        sgxlkl_trace_ignored_syscall = 1;
-        sgxlkl_trace_unsupported_syscall = 1;
-    }
-
-    if (getenv_bool("SGXLKL_TRACE_LKL_SYSCALL", 0))
-        sgxlkl_trace_lkl_syscall = 1;
-
-    if (getenv_bool("SGXLKL_TRACE_INTERNAL_SYSCALL", 0))
-        sgxlkl_trace_internal_syscall = 1;
-
-    if (getenv_bool("SGXLKL_TRACE_IGNORED_SYSCALL", 0))
-        sgxlkl_trace_ignored_syscall = 1;
-
-    if (getenv_bool("SGXLKL_TRACE_UNSUPPORTED_SYSCALL", 0))
-        sgxlkl_trace_unsupported_syscall = 1;
-
-    if (getenv_bool("SGXLKL_TRACE_REDIRECT_SYSCALL", 0))
-        sgxlkl_trace_redirect_syscall = 1;
-
-    if (getenv_bool("SGXLKL_TRACE_MMAP", 0))
-        sgxlkl_trace_mmap = 1;
-
-    if (getenv_bool("SGXLKL_TRACE_SIGNAL", 0))
-        sgxlkl_trace_signal = 1;
-
-    if (getenv_bool("SGXLKL_TRACE_THREAD", 0))
-        sgxlkl_trace_thread = 1;
-
-    if (getenv_bool("SGXLKL_TRACE_DISK", 0))
-        sgxlkl_trace_disk = 1;
-
```

Comment:
Propagating the fields from the config structure to the globals here would dramatically reduce the size of this diff.

Reply:
I already wrote the diff... it would dramatically increase my workload if I had to type in again :) 

As above, I want the config to be in one place, not distributed across a bunch of globals in different files.